.openshift-ci/jobs/integration-tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM pipeline:src
+FROM quay.io/stackrox-io/apollo-ci:collector-0.3.44-1-gb00ffc52af
```

Comment:
It could, but after working in this PR I've come to the conclusion we made a terrible mistake by using things like `pipeline` and `replaced-by-osci` in our dockerfiles, the reason being that OSCI can properly replace them when running there without any problem and if we use ACTUAL, PROPER images in the dockerfiles we can run the same builds locally with relative ease.

Reply:
I'll replace the apollo-ci image, the images were drivers are copied from are currently not built locally and will require some extra work to get them working, so we can probably leave it for a follow up, does that make sense to you?

Reply:
Yep sounds reasonable to me!